src/validations/user.validation.js

```diff
@@ -179,6 +180,58 @@ class UserValidation {
     }
     next();
   }
+
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {res} res
+   * @param {next} next
+   * @returns {validation} This function validate view comments route
+   * @memberof UserValidation
+   */
+  static async validateViewComment(req, res, next) {
+    const page = parseInt(req.query.page, 10);
+    const limit = parseInt(req.query.limit, 10);
+
+    const schema = Joi.object({
+      tripId: Joi.number().greater(0).required(),
```

Comment:
Since #56 has been merged, there should be no further use of `tripId` instead use `requestId`